el-patch.el

```diff
@@ -335,12 +335,13 @@ be defined permanently."
 (defun el-patch--classify-definition-type (type)
   "Classifies a definition TYPE as a `function' or `variable'.
 TYPE is a symbol `defun', `defmacro', etc."
-  (pcase type
-    ((or 'defun 'defmacro 'defsubst 'define-minor-mode)
-     'function)
-    ((or 'defvar 'defconst 'defcustom)
-     'variable)
-    (_ (error "Unexpected definition type %S" type))))
+  (let ((classify (assoc type el-patch-deftype-alist)))
+    (if classify
+        (pcase (nth 2 classify)
+          ('el-patch-classify-function 'function)
+          ('el-patch-classify-define-minor-mode 'function)
+          ('el-patch-classify-variable 'variable))
+      (error "Unexpected definition type %S" type))))
```

Comment:
I like to put the `%S` in quotes, like `` `%s' ``.

Reply:
Whoops, I guess I changed my mind about formatting guidelines sometime since I created this package in January 2017.